jhrasco-ct/dotTheory-TypeScript
# JS-719: Getting Started with TypeScript

## Related Tickets

- [JS-719: Getting Started with TypeScript](https://codeandtheory.atlassian.net/browse/JS-719)

## Description

Completed the project setup for the "The Course Project Setup" topic.

## Validation

1. Open the project folder.
2. Open the `index.html` file.
3. Open the browser's console.
4. Verify that the log statement is shown.

## Screenshots

<img width="742" alt="Screenshot 2023-05-15 at 6 10 05 PM" src="https://github.com/jhrasco-ct/dotTheory-TypeScript/assets/89834420/0b5ea1d1-5c1d-4f97-a20d-65572bd54cbb">
